project/make/validate-toml

```diff
@@ -9,9 +9,7 @@ unset IFS
 badFiles=()
 for f in "${files[@]}"; do
 	# we use "git show" here to validate that what's committed is formatted
-	if [ "$(git show "$VALIDATE_HEAD:$f" | tomlv)" ]; then
-		badFiles+=( "$f" )
-	fi
+	tomlv $f || badFiles+=( "$f" )
```

Comment:
Ah it's because I was on my other computer facepalm

On Wednesday, February 18, 2015, Tianon Gravi <notifications@github.com>
wrote:

> In project/make/validate-toml
> <https://github.com/docker/docker/pull/10872#discussion_r24970663>:
>
> > @@ -9,9 +9,7 @@ unset IFS
> >  badFiles=()
> >  for f in "${files[@]}"; do
> >  	# we use "git show" here to validate that what's committed is formatted
> > -	if [ "$(git show "$VALIDATE_HEAD:$f" | tomlv)" ]; then
> > -		badFiles+=( "$f" )
> > -	fi
> > +	tomlv $f || badFiles+=( "$f" )
>
> (or your expanded if which is probably easier to read and understand)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10872/files#r24970663>.
>